.devcontainer/Dockerfile

```diff
@@ -5,4 +5,4 @@ FROM nats:latest as nats
 FROM mcr.microsoft.com/vscode/devcontainers/go:0-${GO_VERSION}-bullseye
 
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive && \
-    apt-get -y install --no-install-recommends bash-completion vim
\ No newline at end of file
+    apt-get -y install --no-install-recommends bash-completion vim && apt-get clean
```

Comment:
nit: devcontainers are ok to leave "bloated" since I may want to install more packages during dev work, a package cache is ok to keep around :)